templates/main.html

```diff
@@ -24,7 +24,7 @@
 <body>
 	<!-- this will be under the whole container -->
 	<div class="container">
-	<!-- <div class="snow"></div> -->
+	<div class="snow"></div> 
```

Comment:
Snow is still broken and both my computers. Can we keep it disabled until we figure this out. Sorry.

Reply:
yes, will change that!